### ISSUE TITLE
What does this plugin do?

### ISSUE THREAD

**Issue by sfilipovic** [jbaronfr/ToTheLeft#1]
Maybe you can make a small gif movie to show what the plugin does. The explanation doesn't actually give me any idea what to expect after installing.

**Comment by jbaronfr** [comment]
Good idea, I left that project aside but definitely can be refreshed with some relevant info ! Thanks for the interest :)
